knowledge-base/actions/endbug/add-and-commit/action-security.yml

```diff
@@ -0,0 +1,11 @@
+name: Add & Commit
+github-token:
+  action-input:
+    input: token
+    is-default: true
+  permissions:
+    contents: write
+    contents-reason: to fetch code
+    contents-if: ${{ with['use-check'] == 'true'}}
```

Comment:
@nasablackberry also please update the condition to `contents-if: ${{ with['push'] != 'false'}}`. view [here](https://github.com/EndBug/add-and-commit/blob/b3fb60fa0604bb9b1514cb5750dc22e18df06995/src/main.ts#L156), [here](https://github.com/EndBug/add-and-commit#pushing)